File: 7-kyu/reverse-factorials/javascript/solution.js
function factorial(n) {
  return n === 1? 1 : n * factorial(n - 1);
}

function reverseFactorial(num) {
  let i = 1;
  while (factorial(i) < num) {
    i++;
  }
  if (factorial(i) > num)
    return 'None';
  else
    return `${i}!`;
}

File: 7-kyu/reverse-factorials/python/solution.py
def factorial(n):
    return 1 if n < 2 else n * factorial(n - 1)

def reverse_factorial(num):
    i = 1
    while factorial(i) < num:
        i += 1
    return 'None' if factorial(i) > num else str(i) + '!'
    